helpers/testrpm.sh

```diff
@@ -18,25 +18,23 @@
 set -e
 
 checkcmd() {
-    command -v $1 >/dev/null 2>&1 || { echo >&2 "This script requires '$1' but it isn't available. Aborting."; exit 1; }
+    command -v "$1" >/dev/null 2>&1 || { echo >&2 "This script requires '$1' but it isn't available. Aborting."; exit 1; }
 }
 
-CUR=$(pwd)
-BASE=$(cd $(dirname $0)/.. && /bin/pwd)
-COPR_REPO=openquake
+COPR_REPO=openquake-stable
 
 checkcmd docker
 
 while (( "$#" )); do
     case "$1" in
         "-h")
             echo "Usage: $0 [-l] [-r]"
-            echo -e "\nOptions:\n\t-l: test RPM locally\n\t-r <reponame>: use a GEM COPR repo provided by <reponame> (default: openquake)"
+            echo -e "\\nOptions:\\n\\t-l: test RPM locally\\n\\t-r <reponame>: use a GEM COPR repo provided by <reponame> (default: openquake)"
```

Comment:
What is the reason to double the escaping char ?

Reply:
Backslash is literal in "\n". Prefer explicit escaping: "\\n". [SC1117]